lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -379,8 +380,40 @@ def fill_change_of_address(change_of_address_source, change_of_address)
         end
       end
 
+      # only needs currentlyHomeless from 'homelessness' source
+      def fill_currently_homeless(source, target)
+        options = source['homelessSituationType']&.strip
+        send_other_description = options == 'OTHER'
+        other_description = source['otherLivingSituation'] || nil
+        target.currently_homeless = Requests::CurrentlyHomeless.new(
+          homeless_situation_options: options,
+          other_description: send_other_description ? other_description : nil
+        )
+      end
+
+      # needs whole `homelessness` object source
+      def fill_risk_of_becoming_homeless(source, target)
+        target.risk_of_becoming_homeless = Requests::RiskOfBecomingHomeless.new(
+          living_situation_options: source['homelessnessRisk']['homelessnessRiskSituationType'],
+          other_description: source['homelessnessRisk']['otherLivingSituation']
+        )
+        target.point_of_contact = source['pointOfContact']['pointOfContactName']
+        primary_phone = source['pointOfContact']['primaryPhone']
+        target.point_of_contact_number = Requests::ContactNumber.new(
+          telephone: primary_phone['areaCode'] + primary_phone['phoneNumber']
+        )
+      end
+
+      def convert_nullable_bool(boolean_value)
+        return 'YES' if boolean_value == true
+        return 'NO' if boolean_value == false
+
+        # placing nil may not be necessary
+        nil
```

Comment:
Nice addition! Could `nil` be considered a falsey value? Is there a specific case for it to be `nil`?

Reply:
the answer here is "i'm not sure", that's why i have it explicitly called out just in case... 😅 
it could be that LH simply does not fill out the question if the value is `nil`, that's why i didn't want it to be falsey.
i am unsure of LH's logic here 😭

Reply:
Ah gotcha. In that case, the existence of the value can be an assumption we make until they tell us otherwise